cluster/libvirt-coreos/util.sh

```diff
@@ -146,6 +162,25 @@ function render-template {
   eval "echo \"$(cat $1)\""
 }
 
+function wait-cluster-readiness {
+  echo "Wait for cluster readiness"
+  local kubectl="${KUBE_ROOT}/cluster/kubectl.sh"
+
+  local timeout=50
+  while [[ $timeout -ne 0 ]]; do
+    nb_ready_minions=$("${kubectl}" get minions -o template -t "{{range.items}}{{range.status.conditions}}{{.kind}}{{end}}:{{end}}" 2>/dev/null | tr ':' '\n' | grep -c Ready || true)
+    echo "Nb ready minions: " $nb_ready_minions
```

Comment:
Elsewhere we print out number ready out of number expected. Doing so here would be nice as well (so you can see how close you are to having the cluster ready).